openshift/4.0/templates/jenkins-slaves/jenkins-slave-generic-template.yaml

```diff
@@ -14,7 +14,6 @@ objects:
     metadata:
       labels:
         build: "${NAME}"
-        role: jenkins-slave
```

Comment:
This role causes Jenkins to overwrite the config map agent configuration with empty defaults. We don't want this behaviour since we are using config maps (default resources cause our build to fail), so I see no issues removing this.